litgpt/scripts/merge_lora.py

```diff
@@ -20,8 +20,8 @@ def merge_lora(
 ) -> None:
     """Merges the LoRA weights with the base model. See `litgpt/finetune/lora.py`.
 
-    Merging happens in-place in the checkpoint directory that is given as input. It also saves
-    a backup file `lit_model.pth.lora` of the trained LoRA weights in case you still need it later.
+    Merging generates a new `lit_model.pth` file by merging the LoRA weights (`lit_model.pth.lora`)
+    with the original checkpoint weights.
```

Comment:
Nit: Maybe just
```suggestion
    Creates a new `lit_model.pth` file by merging the LoRA weights (`lit_model.pth.lora`)
    with the original checkpoint weights.
```
To avoid word repetation?